pkg/apis/workflow/v1alpha1/item.go

```diff
@@ -20,11 +20,10 @@ const (
 // Item expands a single workflow step into multiple parallel steps
 // The value of Item can be a map, string, bool, or number
 //
-// +protobuf=true
 // +protobuf.options.(gogoproto.goproto_stringer)=false
-// +k8s:openapi-gen=true
+// +kubebuilder:validation:Type=object
```

Comment:
this is actually not fixable, because `Item` can be anything, but `code-generator` mandates a single type only, we must choose the best/least worst type.